src/main/groovy/com/avioconsulting/mule/linter/model/ConfigurationFile.groovy

```diff
@@ -44,6 +93,21 @@ class ConfigurationFile extends ProjectFile {
         List<GPathResult> components = configXml.depthFirst().findAll {
             it.name() == componentType && it.namespaceURI() == namespace
         }
+        comps.each { comp ->
+            Map<String, String> atts = [:]
+            comp[0].attributes.each {
+                atts.put(it.key, it.value)
+            }
+            componentList.add(new MuleComponent(comp.name(), atts))
+            //TODO this doesn't account for child components...
+        }
+    }
+
+    List<MuleComponent> containsConfiguration(String componentType, String namespace) {
```

Comment:
I think this may be part of the above merge issue.

Reply:
fixed